testing/functional/tests/cypress.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This file does not look formatted. Am I missing something?

Reply:
Changes have been done.